extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+<!-- omit in toc -->
 # 3DTILES_implicit_tiling
 
 **Version 0.0.0**, December 1, 2020
```

Comment:
What date should we put here? We technically finished the draft on the 11th, but we could also wait until the day we merge this PR. 

Just remember to update this on the landing page too.